.github/workflows/github.yml

```diff
@@ -8,7 +8,7 @@ on:
     pull_request:
 env:
   # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
-  BUILD_TYPE: Release
+  BUILD_TYPE: RelWithDebInfo
```

Comment:
Could you make debug symbols artifact so that in case of crash we can take them and crashdump. Without symbols crash dump is useless

Reply:
Just gather all pdb files to zip archive.

Reply:
@nullkiller I can. I will do then.

Reply:
Probably we can have it as a separate review

Reply:
I mean debug files. *.pdb Probably it is not needed now because we have mxe with debug symbols already onboard

Reply:
Only MXE is published to downloads so only it is used by Windows community

Reply:
@nullkiller maybe. After this is merged, I want to do 2 new CI jobs. Can I?

Reply:
@nullkiller ok. But this is only after this PR will be merged)